airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -14,21 +14,30 @@ dependencies {
     implementation project(':airbyte-integrations:bases:base-java')
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 
-    implementation('org.apache.spark:spark-sql_2.13:3.3.0') {
+    implementation('org.apache.spark:spark-sql_2.13:3.3.1') {
```

Comment:
this is so that the same Spark version is used that Iceberg 1.1.0 is built for: https://github.com/apache/iceberg/blob/apache-iceberg-1.1.0/spark/v3.3/build.gradle#L20